xcm/xcm-builder/src/universal_exports.rs

```diff
@@ -160,10 +160,7 @@ impl<Bridges: ExporterFor, Router: SendXcm, UniversalLocation: Get<InteriorMulti
 		// export for free. Common-good chains will typically be afforded this.
 		let message =
 			Xcm(vec![ExportMessage { network: remote_network, destination: remote_location, xcm }]);
-		let (v, mut cost) = validate_send::<Router>(bridge, message)?;
-		if let Some(payment) = maybe_payment {
-			cost.push(payment);
-		}
+		let (v, cost) = validate_send::<Router>(bridge, message)?;
 		Ok((v, cost))
```

Comment:
I was about to comment that we can actually remove the `Ok((v, cost))` and replace the 2 lines with just:

```rust
validate_send::<Router>(bridge, message)
```

Reply:
Oh right. I missed it. I'll open another small PR for this.